telosnetwork/telos-wallet
# changing user dropdown menu styling

# Fixes #243

## screenshot
![image](https://user-images.githubusercontent.com/4420760/231641425-6868b358-7879-4638-8743-6b8bf55e9ff3.png)

## Test scenarios
- go to [this link](https://deploy-preview-250--wallet-staging.netlify.app/)
- authenticate with Metamask
- press the user info dropdown button.
  - the style of the dropdown should fit the design
  - the dark mode should not appear
